packages/core/src/sagas/sendMessageBackToPostActivitySaga.js

```diff
@@ -0,0 +1,30 @@
+import {
+    put,
+    take
+  } from 'redux-saga/effects';
+
+  import whileConnected from './effects/whileConnected';
+
+  import { SEND_MESSAGE_BACK } from '../actions/sendMessageBack';
+  import postActivity from '../actions/postActivity';
+
+  export default function* () {
+    yield whileConnected(function* () {
+      for (;;) {
```

Comment:
This seems weird to me, however I have not used `redux-saga` before. Would a `while (true)` be the same thing?

Reply:
Yes, this is actually the first time I'm using `for (;;)`, at @compulim's suggestion. It also keeps code parity with the rest of the project, so I figure keep as is. LMK if I should change. :)

Reply:
Few advantages for `for (;;)`:
- Code parity
- Shorter after uglify (vs. `while(1)` is 8-character)
- `while` is doing a truthy check, and `for` is just a `JMP` operation (although on JVM side, it might optimize this)